client/app/components/Input/__tests__/InputPassword.spec.jsx

```diff
@@ -0,0 +1,30 @@
+// @flow
+import { mount } from 'enzyme';
+import { InputMocks } from '../../../mocks/InputMocks';
+import { shallow } from 'enzyme';
+import React from 'react';
+import { InputPassword } from '../InputPassword';
+
+const text = 'some-password-text';
+const component = InputMocks.createInput(InputMocks.inputPasswordProps);
+const input = `input#${InputMocks.inputPasswordProps.id}`;
+
+  describe('on mouse click', () => {
+    it('toggles correctly', () => {
+      const wrapper = mount(component);
+
+      wrapper
+        .find('i')
+        .simulate('click');
+      expect(wrapper.find('input').props().value).toEqual(text);  //some-password-text
+      expect(wrapper.find('input').props().type).toEqual("password");
+      expect(wrapper.find('i').props().className).toEqual("fa fa-eye-slash");
+
+      wrapper
+        .find('i')
+        .simulate('click');
+      expect(wrapper.find('input').props().value).toEqual(text);
+      expect(wrapper.find('input').props().type).toEqual("text");
+      expect(wrapper.find('i').props().className).toEqual("fa fa-eye");
```

Comment:
We should also test for typing in a password value and toggling this show button. Then we can test for `type="text"` vs. `type="password"`.